engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -570,7 +569,7 @@ private boolean renderLightComponent(LightComponent lightComponent, Vector3f lig
             program.setFloat3("lightColorAmbient", lightComponent.lightColorAmbient.x, lightComponent.lightColorAmbient.y, lightComponent.lightColorAmbient.z, true);
 
             program.setFloat4("lightProperties", lightComponent.lightAmbientIntensity, lightComponent.lightDiffuseIntensity,
-                    lightComponent.lightSpecularIntensity, lightComponent.lightSpecularPower, true);
+                    0.0f, lightComponent.lightSpecularPower, true);
```

Comment:
mmmm...  please avoid magic numbers, especially when the setFloat4 function would give no chance of easily finding out what the fourth parameter passed in means. Perhaps make a constant at the beginning of the file?